test/Libraries/TestServices/GeometricTestBase.cs

```diff
@@ -1,10 +1,11 @@
 ﻿using System;
```

Comment:
Unit tests now rely on `DynamoShapeManager.Preloader` for its geometry preloading activities.